src/components/ListDetail/ListDetail.module.css

```diff
@@ -124,13 +137,21 @@
   color: var(--eds-theme-color-text-disabled); /* 2 */
   cursor: pointer;
   transition: color var(--eds-anim-fade-quick) var(--eds-anim-ease),
-    box-shadow var(--eds-anim-fade-quick) var(--eds-anim-ease),
-    background-color var(--eds-anim-fade-quick) var(--eds-anim-ease);
+  box-shadow var(--eds-anim-fade-quick) var(--eds-anim-ease),
+  background-color var(--eds-anim-fade-quick) var(--eds-anim-ease);
+  align-items: center;
+  justify-content: space-between;
+  max-width: var(--eds-l-sidebar-width);
 
   &:focus-visible {
     @mixin focus;
   }
 
+  @media all and (max-width: $eds-bp-lg) {
```

Comment:
on < lg viewports, all list items (including unselected) are bold and text neutral